### ISSUE TITLE
feature: option to select search provider: DocSearch or AppSearch

### ISSUE THREAD

**Issue by liyasthomas** [nuxt-themes/docus#965]
Recently in-app text search feature with the `fuse` lib has been implemented. But this introduced a few minor UX issues.

**Current behavior:**
- Currently, Algolia DocSearch and AppSearch components are enabled whenever there's an `algolia` property present in `runtimeConfig`. There's no option to disable either of them. I wish there was a flag to enable/disable both search functionalities directly from `nuxt.config.js`.

- <kbd>CMD</kbd> + <kbd>K</kbd> keyboard shortcut brings up both the DocSearch component by Algolia and Fuse search component. Keyboard binding should only bring up one (active) search component.

<img width="1470" alt="Screenshot 2023-08-01 at 4 15 13 PM" src="https://github.com/nuxt-themes/docus/assets/10395817/86834a4a-361d-467b-bad5-82f49f173484">

**Comment by bdrtsky** [comment]
How it happened that you have both `AppSearch` and `DocsSearch`?

**Comment by liyasthomas** [comment]
Here's the link to the deployment: https://docs.hoppscotch.io
GitHub repo: https://github.com/hoppscotch/docs
package.json: https://github.com/hoppscotch/docs/blob/main/package.json
nuxt.config.json: https://github.com/hoppscotch/docs/blob/main/nuxt.config.ts

**Comment by Atinux** [comment]
Would you prefer that we disable Fuse if you have Algolia @liyasthomas ?

**Comment by liyasthomas** [comment]
Yes. Imo, if algolia keys are present, in-app search should be disabled. For backward compatibility purposes introducing Fuse should not disable Algolia search by default. And for those who hasn't setup Algolia DocSearch, Fuse will be enabled as well.

**Comment by Atinux** [comment]
Completely agree and very sorry about this, will land a fix ASAP

**Comment by liyasthomas** [comment]
Verified v1.14.6 resolved this issue. Thanks a lot, @Atinux for the fast response 🙏.
